programs/stopwatch_example/Makefile

```diff
@@ -0,0 +1,10 @@
+CFLAGS=-Wall -Werror -pedantic -std=gnu99
+LDFLAGS=-rdynamic -lproc -ldl -lm -L /usr/local/lib
```

Comment:
You shouldn't need to hardcode `-L /usr/local/lib` ... if your computer doesn't automatically search there I can show you how to fix it.

Reply:
Yes, I'd appreciate that!